[Traffic Scotland] Add mapfile for Traffic Scotland

MorayMySoc opened the pull request:
Combine Scotland/England road network to single asset layer

https://github.com/mysociety/societyworks/issues/3442

dracos reviewed (approved):
Looks fine, as discussed could go in highways layer if easier

MorayMySoc committed:
[Highways][Traffic Scotland] Add connection to combined Highways

National Highways asset is now combined with Traffic Scotland
and serves a descriptor field brought in by Traffic Scotland.

https://github.com/mysociety/societyworks/issues/3442

MorayMySoc commented:
Moved to Highways layer as using a combined shape file.

MorayMySoc commented:
Added a class for https://report.nationalhighways.co.uk/reports/National+Highways

![Screenshot from 2025-06-04 18-18-39](https://github.com/user-attachments/assets/bad675da-b677-4fde-a0e3-2b8d8d9e0c12)

davea commented:
As we only show the WMS layer on the National Highways site, could we hide the Scotland roads entirely? They're still very apparent in that screenshot.

dracos commented:
Was probably my suggestion to make them grey as opposed to green/blue for the 'active' NH roads. If it's not possible to make them transparent (I'd try "#00000000", I think it might handle RGBA), I'd be happy with them being a lighter grey also.
